third_party/move/move-model/bytecode/src/stackless_bytecode.rs

```diff
@@ -1201,6 +1210,31 @@
                     field_env.get_name().display(struct_env.symbol_pool())
                 )?;
             },
+            GetVariantField(mid, sid, variants, targs, offset) => {
+                assert!(!variants.is_empty());
```

Comment:
Better not an assertion in debug print code, what this is here.

Reply:
I directly copied this from the branch `BorrowVariantField`: https://github.com/aptos-labs/aptos-core/blob/main/third_party/move/move-model/bytecode/src/stackless_bytecode.rs#L1163. Do we want to remove both of them?

Reply:
Yeah lets remove it there too

Reply:
done